bitset.go

```diff
@@ -895,8 +895,22 @@ func (b *BitSet) WriteTo(stream io.Writer) (int64, error) {
 	}
 
 	// Write set
-	err = binary.Write(stream, binaryOrder, b.set)
-	return int64(b.BinaryStorageSize()), err
+	// current implementation of bufio.Writer is more memory efficient than
+	// binary.Write for large set
+	writer := bufio.NewWriter(stream)
+	var item = make([]byte, 8)     // for serializing uint64
+	var n = binary.Size(uint64(0)) // number of bytes written
```

Comment:
That seems odd, doesn't it? You assume that the size is 8, and then you next compute it? Do I get this right?